services/data_sources/json_build_data_source.rb

```diff
@@ -94,8 +94,13 @@ def list_builds(project: nil)
       end
 
       most_recent_builds = build_files.map do |build_path|
-        build_object_hash = JSON.parse(File.read(build_path))
-        build = Build.from_json!(build_object_hash)
+        begin
+          build_object_hash = JSON.parse(File.read(build_path))
+          build = Build.from_json!(build_object_hash)
+        rescue => ex
+          logger.debug(ex.to_s)
```

Comment:
don't need `.to_s` our logger does stuff with it. If it doesn't do what you think it should, let's update the logger to handle printing exceptions 😺

Reply:
I know it doesn't, I assume though that we do want to be consistent when it comes to data types for parameters as much as possible.